src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -1,33 +1,41 @@
 package edu.kis.vh.nursery;
 
-public class defaultCountingOutRhymer {
+public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int minusOne = -1;
+    private static final int eleven = 11;
+    private static final int numLength = 12;
+    private int[] NUMBERS = new int[numLength];
 
-    public int total = -1;
+    public int getTotal() {
+        return total;
+    }
+
+
+    public int total = minusOne;
 
-    public void countIn(int in) {
+    protected void countIn(int in) {
         if (!isFull())
             NUMBERS[++total] = in;
     }
 
-        public boolean callCheck() {
-            return total == -1;
+        protected boolean callCheck() {
+            return total == minusOne;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.